### PR TITLE
L10n support with english default

### PR DESCRIPTION
**Pull request by Wulfheart**
Refers to #12.

### CODE REVIEW

File: stubs/auth/resources/lang/en/auth.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    'verify' => [
+        'heading' => 'Verify your email address',
+        'subtitle' => [
+            'pre' => 'Or',
+            'link' => 'sign out',
+        ],
+        'resent_verification_link' => 'A fresh verification link has been sent to your email address.',
+        'check_email_for_link' => 'Before proceeding, please check your email for a verification link.',
+        'email_not_received' => [
+            'pre' => 'If you did not receive the email, ',
```
Comment (imliam) [inline stubs/auth/resources/lang/en/auth.php:61]: I get why this is done, but it doesn't necessarily work to have "pre" and "link" text separately like this. It works for what the current English text is, but that won't be the case for every language.

What I prefer doing for things like this is to use parameters where I can, which lets the translator move it about and put text before or after the link if they need to:

```
'email_not_received' => 'If you did not receive the email, :link.',
'email_not_received_link_text' => 'click here to request another',
```

That does make it a bit more awkward to handle how to actually put the anchor HTML in there though.

Thoughts?

Comment (Wulfheart) [reply stubs/auth/resources/lang/en/auth.php:61]: This doesn't feel right to me but I get your point.

Do you have any experience in doing this? I'm quite clueless on this right now.

**Comment by dakira** [comment]
I would prefer the solution in #14. Translators usually use json translations and existing translation files already work with the other solution.

**Comment by Wulfheart** [comment]
I don't work on it anymore. Therefore I'm closing this.

**Pull request closed without merging**